x-pack/plugins/triggers_actions_ui/kibana.json

```diff
@@ -4,7 +4,7 @@
   "server": false,
   "ui": true,
   "optionalPlugins": ["alerts", "alertingBuiltins"],
-  "requiredPlugins": ["management", "charts", "data"],
+  "requiredPlugins": ["management", "charts", "data", "kibanaReact"],
```

Comment:
It supports [this dependency](https://github.com/elastic/kibana/blob/d2ae16053d6eec8587813de81f019da5a1b22c47/x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/geo_threshold/query_builder/util_components/single_field_select.tsx#L17) which allows us to display the relevant icon for the field being selected. It's arguably not critical but it does make for a better user experience. If you feel strongly, let me know, it can be removed!

Reply:
Oh. I didn't find it, so if you have this usage than it is OK  👍 - we are planning to use 'kibanaReact' soon as well.